### ISSUE TITLE
Feature request for Scrap Broker

### ISSUE THREAD

**Issue by TurduckenMcNugget** [Jaliborc/Scrap#218]
TL:DR - I would like the broker to destroy only the current least valuable item scheduled to be sold instead of all junk items. Sometimes you only want to free up a few bag slots and not throw away everything to be sold.

![Broker_Garbage](https://github.com/Jaliborc/Scrap/assets/42200577/b8ca4bae-abf0-4ebe-be62-7c263b16caf4)

I'm looking to finally move on from an old abandoned addon called Broker_Garbage and yours looks like a good candidate. This request is to ask if you would add functionality to the broker part to work like the other addon did. 

The broker label would show you your current free bag space, the name of your currently least valuable junk item stack, the stack count, and the money that you would get from the vendor by selling it. The tooltip would show a list with the 12 (I think) least valuable junk stacks sorted by vendor value.  If you shift-clicked the broker lable, it would destroy the least valuable, freeing up a bag slot and changing to show the next least valuable. Ctrl+click would instead mark the item as not junk remove it from the list. 

This was nice because if your bags would full you could destroy cheap stuff to free up space, but also stop when the value was high enough that you wouldn't want to destroy the item in favor of the thing you would then be able to pick up.

I've attached a screenshot of the tooltip with a couple of junk items in my inventory. So you can get a better idea of what it is doing. Note, the tooltip also has some stuff about "Vendor Prices"; the addon let you specify custom valuations for items (what you think you might get off the auction house for it, for example) but I'm not asking for that. Just more control over the process of destroying junk when I'm low on bag space.
